Build/Remotion.Server.build

```diff
@@ -50,17 +50,21 @@
   <Target Name="TestBuild_Configuration">
     <PropertyGroup>
       <Platforms>x86</Platforms>
-      <DatabaseSystems>SqlServer2008R2</DatabaseSystems>
-      <Browsers>Chrome</Browsers>
+      <DatabaseSystems>NoDB</DatabaseSystems>
+      <Browsers>NoBrowser</Browsers>
+      <ExecutionRuntimes>Win_NET48+EnforcedLocalMachine</ExecutionRuntimes>
```

Comment:
We can remove EnforcedLocalMachine as well since there's nothing that can't run inside docker
```suggestion
      <ExecutionRuntimes>Win_NET48</ExecutionRuntimes>
```